packages/neon/neon/lib/src/platform/ios.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:meta/meta.dart';
+import 'package:neon/src/platform/platform.dart';
+import 'package:neon/utils.dart';
+import 'package:path/path.dart' as p;
+import 'package:path_provider/path_provider.dart';
+import 'package:permission_handler/permission_handler.dart';
+
+@immutable
+@internal
+class IosNeonPlatform implements NeonPlatform {
+  const IosNeonPlatform();
+
+  @override
+  bool get canUseCamera => true;
+
+  @override
+  bool get canUsePushNotifications => false;
+
+//don't know what this means
```

Comment:
please remove this comment.
This means whether the platform is supported by https://pub.dev/packages/quick_actions wich is true for iOS.
quick_actions are the actions you get when long pressing an app icon on the home screen (at least that's what it is on Android)